apps/docs/src/routes/docs/styles/styled-components.mdx

```diff
@@ -148,7 +148,7 @@ const StyledButton = hope("button", {
 
 ## Using style props, `sx` and the `as` prop
 
-Like others Hope UI components, _styled components_ supports style props, `sx` and the `as` prop to change the rendered element.
+Hope UI components, like others, support style props, sx, and the as prop to change the rendered element.
```

Comment:
I'm not sure if the sentence is correct.

What I wanted to say is that :

Like others ready made Hope UI components, the one you create with the _styled component_ API supports style props, sx, and the as prop

Reply:
Oh, yes, I see. I didn't understand it right. I changed it back to how it was and made a slight change in grammar